src/components/helpdesk/uc/helpdesk.facade.js

```diff
@@ -0,0 +1,15 @@
+const helpdeskUC = require('./helpdesk.uc');
+
+class HelpdeskFacade {
+	setup(app) {
+		this.app = app;
+	}
+
+	async deleteUserData(userId) {
+		return helpdeskUC.deleteProblemsForUser(userId);
```

Comment:
https://github.com/hpi-schul-cloud/schulcloud-server/blob/develop/src/components/user-group/uc/classes.facade.js#L14 here, the facade should return only the uc method executed. 
```suggestion
		return helpdeskUC.deleteProblemsForUser();
```

Reply:
but the usecase method requires a userId?